Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -97,24 +97,24 @@
   private static final long UPDATE_FILES_TIMEOUT = 300;
 
   private static final long CHECK_CANCELLED_PERIOD = 10;
-  private static final long SEND_REQUEST_TIMEOUT = TimeUnit.SECONDS.toMillis(1);
-  private static final long EDIT_FORMAT_TIMEOUT = TimeUnit.SECONDS.toMillis(3);
+  private static final long SEND_REQUEST_TIMEOUT = TimeUnit.SECONDS.toMillis(10);
```

Comment:
As a drive by comment, we may want to normalize all these to `TimeUnit.MILLISECONDS.toMillis()` in order to make it easier to compare between entries? Some here use seconds and some milliseconds, and its easy to mistake the two when scanning down the list.

Reply:
This seems like a very good idea.

We definitely don't want, for instance, GET_SUGGESTIONS_TIMEOUT = 100 Seconds.